Tests/js/views/ez-navigationhubview.html

```diff
@@ -83,9 +103,25 @@
         filter: loaderFilter,
         modules: {
             "ez-navigationhubview": {
-                requires: ['ez-templatebasedview', 'ez-navigationitemview', 'event-tap', 'node-screen', 'node-style', 'event-outside', 'ez-height-change'],
+                requires: ['ez-templatebasedview', 'ez-navigationitemview', 'event-tap', 'node-screen', 'node-style', 'event-outside', 'ez-height-change', 'ez-userprofileview'],
                 fullpath: "../../../Resources/public/js/views/ez-navigationhubview.js"
             },
+            "ez-userprofileview": {
```

Comment:
you should try to limit the dependencies as much as possible.
Actually, this is one of the reason why the sub-views are stored in attributes with a `valueFn` (like `userProfileView` in the case of the navigation hub view). Because of this, in the test to remove the need of `ez-userprofileview`, you can write something like:

```js
setUp: function () {
    this.userProfileView = new Y.View();
    this.view = new Y.eZ.NavigationHubView({
        userProfileView: this.userProfileView,
        // rest of the needed config...
    });
}
```

it's also possible to use a mock instead but in the case of a View that's usually a bit painful. Also, depending on how the sub-view is used, it's also common to dynamically create a fake implementation of the sub-view, this gives something like:

```js
setUp: function () {
    var UserProfileView = Y.Base.create('userProfileView', Y.View, [], {
            /* some needed methods */
        }, {/* even some ATTRS */});
    this.userProfileView = new UserProfileView();
    this.view = new Y.eZ.NavigationHubView({
        userProfileView: this.userProfileView,
        // rest of the needed config...
    });
}
```